lib/Backend/GlobOpt.cpp

```diff
@@ -9585,6 +9584,32 @@ GlobOpt::TypeSpecializeInlineBuiltInBinary(IR::Instr **pInstr, Value *src1Val, V
             break;
         }
 
+        case Js::OpCode::InlineMathPow:
+        {
+#ifndef _M_ARM
```

Comment:
Yes Can we fix LowerMd::GenerateFastInlineBuiltInCall for ARM as well?

---
In reply to: [57507152](https://github.com/Microsoft/ChakraCore/pull/644#discussion_r57507152) [](ancestors = 57507152)